WORKSPACE

```diff
@@ -79,222 +81,256 @@ load(
+    "libnettle6",
+    "libp11-kit0",
+    "libffi6",
+    "libtasn1-6",
+    "libsasl2-modules-db",
+    "libgcrypt20",
+    "libgpg-error0",
+    "libacl1",
+    "libattr1",
+    "libselinux1",
+    "libpcre3",
+    "libbz2-1.0",
+    "liblzma5",
+]
+
+PACKAGES_S390X_DEBIAN9 = [pkg for pkg in PACKAGES_DEBIAN9 if pkg != "libunwind8"]
+
 [
     dpkg_list(
         name = "package_bundle_" + arch + "_debian9",
-        packages = [
-            "libc6",
-            "base-files",
-            "ca-certificates",
-            "openssl",
-            "libssl1.0.2",
-            "libssl1.1",
-            "libbz2-1.0",
-            "libdb5.3",
-            "libffi6",
-            "libncursesw5",
-            "liblzma5",
-            "libexpat1",
-            "libreadline7",
-            "libtinfo5",
-            "libsqlite3-0",
-            "mime-support",
-            "netbase",
-            "readline-common",
-            "tzdata",
-
-            #c++
-            "libgcc1",
-            "libgomp1",
-            "libstdc++6",
-
-            #java
-            "zlib1g",
-            "libjpeg62-turbo",
-            "libpng16-16",
-            "liblcms2-2",
-            "libfreetype6",
-            "fonts-dejavu-core",
-            "fontconfig-config",
-            "libfontconfig1",
-            "libuuid1",
-            "openjdk-8-jre-headless",
-            "openjdk-8-jdk-headless",
-            "openjdk-11-jre-headless",
-            "openjdk-11-jdk-headless",
-            "libc-bin",
-
-            #python
-            "libpython2.7-minimal",
-            "python2.7-minimal",
-            "libpython2.7-stdlib",
-            "dash",
-            "libc-bin",
-
-            #python3
-            "libmpdec2",
-            "libpython3.5-minimal",
-            "libpython3.5-stdlib",
-            "python3.5-minimal",
-
-            #dotnet
-            "libcurl3",
-            "libgssapi-krb5-2",
-            "libicu57",
-            "liblttng-ust0",
-            "libssl1.0.2",
-            "libunwind8",
-            "libuuid1",
-            "zlib1g",
-            "curl",
-            "libcomerr2",
-            "libidn2-0",
-            "libk5crypto3",
-            "libkrb5-3",
-            "libldap-2.4-2",
-            "libldap-common",
-            "libsasl2-2",
-            "libnghttp2-14",
-            "libpsl5",
-            "librtmp1",
-            "libssh2-1",
-            "libkeyutils1",
-            "libkrb5support0",
-            "libunistring0",
-            "libgnutls30",
-            "libgmp10",
-            "libhogweed4",
-            "libidn11",
-            "libnettle6",
-            "libp11-kit0",
-            "libffi6",
-            "libtasn1-6",
-            "libsasl2-modules-db",
-            "libgcrypt20",
-            "libgpg-error0",
-            "libacl1",
-            "libattr1",
-            "libselinux1",
-            "libpcre3",
-            "libbz2-1.0",
-            "liblzma5",
-        ],
+        packages = PACKAGES_DEBIAN9,
```

Comment:
I think it would actually simplify the logic to make this:
```
   packages = PACKAGES_DEBIAN9 + if arch in BASE_ARCHITECTURES: ["libunwind8"] else [],
```

You could even leave `PACKAGES_DEBIAN9` inlined here, and make this diff *much* smaller.